### ISSUE TITLE
It shown /device:GPU:0 on console log although I have no Nvidia GPU

### ISSUE THREAD

**Issue by ttpro1995** [tensorflow/nmt#280]
I follow the seq2seq guide https://www.tensorflow.org/tutorials/seq2seq
When I run 

python -m nmt.nmt \
    --src=vi --tgt=en \
    --vocab_prefix=/tmp/nmt_data/vocab  \
    --train_prefix=/tmp/nmt_data/train \
    --dev_prefix=/tmp/nmt_data/tst2012  \
    --test_prefix=/tmp/nmt_data/tst2013 \
    --out_dir=/tmp/nmt_model \
    --num_train_steps=12000 \
    --steps_per_stats=100 \
    --num_layers=2 \
    --num_units=128 \
    --dropout=0.2 \
    --metrics=bleu

I noted that is shown /device:GPU:0 on terminal 

![terminal](https://i.imgur.com/ji1c9VG.png) 

However, I don't remember buying a GPU for my PC. I do check with 

```
>>> import tensorflow as tf
>>> tf.test.gpu_device_name()
''
```
=> no GPU detected.
